### ISSUE TITLE
[BUG] Editor styles overwriting global styles

### ISSUE THREAD

**Issue by chrassendk** [Darginec05/Yoopta-Editor#186]
### Has this bug been raised before?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Description

When embedding the editor the local styles for the editor is globally applied.
<img width="665" alt="image" src="https://github.com/Darginec05/Yoopta-Editor/assets/10469954/b7aee9a3-2ff1-43e4-bdb4-d7cc9f0de9fe">. https://github.com/Darginec05/Yoopta-Editor/blob/cd8c25f695296c87ae27138330038d931301f663/packages/core/editor/src/styles.css

### Steps to Reproduce

1. Add editor.
2. Tailwind buttons e.g. loose their border

### Environment

- OS:
- Browser:

### Screenshots

<img width="665" alt="image" src="https://github.com/Darginec05/Yoopta-Editor/assets/10469954/f01df34e-9e1f-4bb5-9d67-73bae5e4f9bc">

### Do you want to work on this issue?

No

**Comment by XavierDK** [comment]
Hi 👋,

Here's a temporary fix that might help you:
```
useEffect(() => {
    document.head.querySelectorAll("style").forEach((style) => {
      if (style.innerHTML.includes("button")) {
        style.innerHTML = style.innerHTML.replace("border:none", "");
      }
    });
  }, []);
```

Hope this helps!

**Comment by chrassendk** [comment]
It will not work on SPA renderings, since it only runs on mount

**Comment by Darginec05** [comment]
Will be fixed in next minor version.
Thanks for reporting

**Comment by Darginec05** [comment]
Fixed in v4.7.0 🎉
